src/nyc_trees/apps/event/views.py

```diff
@@ -29,8 +29,10 @@ def add_event(request):
     is_valid = _process_event_form(form, request)
 
     if is_valid:
+        base_url = reverse(
+            'group_edit', kwargs={'group_slug': request.group.slug})
         return HttpResponseRedirect(
-            reverse('group_edit', kwargs={'group_slug': request.group.slug}))
+            '%s#%s' % (base_url, GROUP_EDIT_EVENTS_TAB_ID))
```

Comment:
I would be in favor of hard-coding `#events` throughout instead of storing it in a constant.